APIBacklog/documentation/MeetingMinutes/MOM-2023-11-02.md

```diff
@@ -0,0 +1,153 @@
+	- In case a WG participant wants to include a point in the agenda (e.g., present a new API proposal), this participant shall ensure the corresponding issue is opened in Github by then. 
+	- Exceptional situations will be treated separately.
+2. To shift API backlog WG conf. calls 1 week, ensuring TSC and API backlog meetings are scheduled in alternate weeks. This action will enable more TSC delegates to join the API backlog WG. Proposals is to schedule biweekly calls:
+	- 2nd Thursday of the month (10-11 CET)
+	- 4th Thursday of the month (16-17 CET)
+
+**Comments during the call**:
+- Consensus with point 1 -> to be communicated offline.
+- No objection with point 2, but will be taken for 7-day mail approval (AP 20231102-01).
+	- Reason is that many regular delegates did not attend the meeting.
+	- Proposed slots may impact American and Asian delegates, so their feedback is important. 
+
+
+## Backlog table updated
+- PR [#331](https://github.com/camaraproject/WorkingGroups/pull/331) ongoing. This is the first step of all the actions identified in [#100](https://github.com/camaraproject/Governance/issues/100). 
+
+## Discussion
+
+| **Agenda  Item**  |  Owner| Description |
+| ----------- | --- | ----------- |
+| **Recent updates & recap** | | - Minutes of last API backlog WG call available [here](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/MeetingMinutes/MOM-2023-10-19.md) <br> - Minutes of last TSC call available [here](https://github.com/camaraproject/Governance/blob/main/documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md). <br> - Device Swap Sub-Project created. <br> - IMEI Fraud can get started in [DeviceIdentifier](https://github.com/camaraproject/DeviceIdentifier) Sub-Project. <br> - Five new APIs were submitted into backlog, all of them coming from Open Gateway Drop #3. Three of them included in today's agenda.| 
+| **Current issues** |  |  |
+| [#271](https://github.com/camaraproject/WorkingGroups/issues/271)  | China Telecom, Huawei| **New API proposal: Site to cloud (S2C) VPN**</br> Updates: <br> - Proposed for drop #3, but unfortunately didn't go shortlisted. </br> - Progress offline among interested parties (API owners, AT&T) with API backlog and TSC mediating. Good discussion and clarification on technical and product scope. <br> - TSC reported offline that they are in position of accepting the proposal, though encourage AT&T and China Telecom aligment within the Sub-Project. <br> - Eric (Vodafone): he reported that TSC gives green light to this Sub-Project, with this decision formally made on 2nd Nov call. <br> - Jose (Telefonica): he asks Pierre (AT&T) whether AT&T would join the Sub-Project, taking guidance from TSC. <br> - Pierre (AT&T): need to align internally with the AT&T colleagues who led the Catalyst, and would come back with a response (AP 20231102-02). <br> - Once the sub-project is approved, the supporters need to provide initial maintainers (AP 20231102-02).|
```

Comment:
You might want to add a note that the TSC on Nov 2nd approved the creation of the sub project. The minutes are still in work, but please go forward to provide the maintainers.

Reply:
Done. Thank you!